src/sonic-yang-models/tests/yang_model_tests/tests_config/feature.json

```diff
@@ -57,7 +62,8 @@
                         "auto_restart": "always_enabled",
                         "has_timer": "false",
                         "has_global_scope": "true",
-                        "has_per_asic_scope": "true"
+                        "has_per_asic_scope": "true",
+                        "set_owner": "lcoal"
```

Comment:
Is this an intentional failure case?

Can there be a test case w/o "set_owner" entry ?

Reply:
Also a test with wrong value for set_owner will be useful.

Reply:
add new test in the latest commit